IOTATransferRestRequestTemplate.py

```diff
@@ -1,28 +1,26 @@
 from __future__ import absolute_import, division, print_function, \
     unicode_literals
 
-import requests
-from iota import *
-from iota import iota
-from iota.adapter.sandbox import SandboxAdapter
 import uuid
 from getpass import getpass as secure_input
-from typing import Optional, Text
 
-from iota import __version__, Iota
+import requests
+from iota import Address, Iota, ProposedTransaction, Tag, TryteString
+from iota.adapter.sandbox import SandboxAdapter
 from iota.crypto.types import Seed
-from six import binary_type, moves as compat, text_type
-from pprint import pprint
+from six import binary_type, moves as compat
+from typing import Optional, Text
 
 
 # this method will register the request by generating a unique id for the request, this will serve as the tag for the
 # sample transaction
 def register_request():
+    # type: () -> uuid.UUID
```

Comment:
Added PEP-484 type hints, just so that everything is consistent.  These work amazingly well in PyCharm; I haven't tested in other IDEs though.